app/views/forums/show.html.erb

```diff
@@ -57,13 +57,8 @@ See COPYRIGHT and LICENSE files for more details.
     header.with_breadcrumbs([{ href: project_overview_path(@project.id), text: @project.name },
                              { href: project_forums_path(@project), text: t(:label_forum_plural) },
                              @forum.name])
-    unless User.current.anonymous?
```

Comment:
This line which checks a condition is removed, I think it was unintended.

Reply:
It is being the check is made here already https://github.com/opf/openproject/pull/17592/files#diff-295f15fbbdfa754df02e2c61f0b0912aaf5179483ae652523ab699fcffce9445L48